charts/pxc-db/values.yaml

```diff
@@ -19,29 +19,34 @@ annotations: {}
 
 operatorImageRepository: percona/percona-xtradb-cluster-operator
 
-crVersion: 1.14.0
+crVersion: 1.15.0
 ignoreAnnotations: []
   # - iam.amazonaws.com/role
 ignoreLabels: []
   # - rack
 pause: false
 # initContainer:
-#   image: ""
+#   image: "perconalab/percona-xtradb-cluster-operator:1.15.0"
```

Comment:
maybe don't mention `perconalab` images, even if it is commented, better use `percona`